### PR TITLE
(feat) adding windows ltsc-2022 support

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

**Review by eoinmcafee00 — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -172,6 +172,57 @@ trigger:
...
+  image: plugins/docker
+  settings:
+    dockerfile: docker/Dockerfile.windows.1809
+    repo: plugins/sftp
+    username:
+      from_secret: docker_username
+    password:
+      from_secret: docker_password
+    auto_tag: true
+    auto_tag_suffix: windows-1809-amd64
```
Comment (donny-dont) [inline .drone.yml:211]: Copy pasta error here

File: .drone.yml [hunk 2]
```diff
@@ -172,6 +172,57 @@ trigger:
...
+- name: build
+  image: golang:1.18
+  environment:
+    CGO_ENABLED: '0'
+  commands:
+  - go build -v -ldflags "-X main.version=" -a -tags netgo -o release/windows/amd64/drone-sftp.exe .
+- name: docker
+  image: plugins/docker
+  settings:
+    dockerfile: docker/Dockerfile.windows.1809
```
Comment (donny-dont) [inline .drone.yml:204]: Copy pasta error here

File: .drone.yml [hunk 3]
```diff
@@ -172,6 +172,57 @@ trigger:
   - refs/tags/**
   - refs/pull/**
 
+
+---
+kind: pipeline
+type: vm
+name: windows-ltsc2022
```
Comment (donny-dont) [inline .drone.yml:179]: Just making a note that the official docker images that provide 2022 images have this as their suffix so this is good to go.

**Review by donny-dont — APPROVED** [review]
Fix the typos and this is good to go.

File: .drone.yml (hunk 1 above)
Comment (tphoney) [reply .drone.yml:211]: im a dumb dumb

**Pull request merged**